### ISSUE TITLE
[Bug] Const layer Constant_2340 has incorrect dimensions in the output data 0

### ISSUE THREAD

**Issue by friend045** [openvinotoolkit/openvino#11598]
**System information (version)**
- OpenVINO => 2022.1
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2019
- Problem classification: Model Conversion
- Framework: Pytorch -> Onnx -> **OpenVino -> Blob** 

**Detailed description**

```
compile_tool -m model.xml -ip U8 -d MYRIAD -VPU_NUMBER_OF_SHAVES 5 -VPU_NUMBER_OF_CMX_SLICES 5
```

When using OpenVino 2022.01 version, I get the following error when converting the model.

```
OpenVINO Runtime version ......... 2022.1.0
Build ........... 2022.1.0-7019-cdb9bec7210-releases/2022/1
Network inputs:
    image : u8 / [...]
Network outputs:
    predy/sink_port_0 : f16 / [...]
    predx/sink_port_0 : f16 / [...]
Const layer Constant_2525 has incorrect dimensions in the output data 0
```
But OpenVino 2021.04 Ver. is fine.

```
Inference Engine:
        IE version ......... 2021.4.2
        Build ........... 2021.4.2-3974-e2a469a3450-releases/2021/4

Network inputs:
    image : U8 / NCHW
Network outputs:
    predx : FP16 / 95
    predy : FP16 / 95
[Warning][VPU][Config] Deprecated option was used : VPU_MYRIAD_PLATFORM
Done. LoadNetwork time elapsed: 5179 ms
```
What is happened?

Also, what does "[Warning][VPU][Config] Deprecated option was used : VPU_MYRIAD_PLATFORM" mean?

Thanks.

**Comment by friend045** [comment]
@jgespino ,

[Sample onnx model](https://drive.google.com/drive/folders/1MfQ3qXMVG3jhwjmk9YBbmlYMI6OtOsVc?usp=sharing)

The following is the Onnx model and model optimizer command.

Onnx to IR
```
mo --input_model ImgxDepth2.onnx --input_shape [1,3,128,128] --scale_values [127.5] --mean_values [1,1,1] --input image --data_type FP16
```

IR to  MyriadX blob

```
compile_tool -m ImgxDepth2.xml -ip U8 -d MYRIAD -VPU_NUMBER_OF_SHAVES 5 -VPU_NUMBER_OF_CMX_SLICES 5
```


Thanks .

**Comment by Iffa-Meah** [comment]
@friend045 ,
we need access to your file. I had requested it.

![image](https://user-images.githubusercontent.com/79643476/166213246-8cd0bf7f-9bdb-4235-ad07-a7436d611317.png)

**Comment by friend045** [comment]
> @friend045 , we need access to your file. I had requested it.
> 
> ![image](https://user-images.githubusercontent.com/79643476/166213246-8cd0bf7f-9bdb-4235-ad07-a7436d611317.png)

@Iffa-Meah ,

Sorry, I was on vacation until yesterday.
 I have opened the permission, please test the URL again.
Thanks.

**Comment by Iffa-Meah** [comment]
Starting from the 2020.4 release, OpenVINO™ supports reading native ONNX models.
You may refer [here](https://docs.openvino.ai/2021.1/openvino_docs_IE_DG_ONNX_Support.html).

I tested your model. The native ONNX does works with benchmark app (CPU).
<img width="479" alt="benchmarkonnx" src="https://user-images.githubusercontent.com/79643476/166405649-a7676c0d-2f5e-414b-81bc-b15e70a9b34c.png">

and the compile tool also worked for the native onnx (CPU)

<img width="655" alt="CPUcompiletool" src="https://user-images.githubusercontent.com/79643476/166405862-f71eabae-735c-497a-8a84-d85aa4f39e74.png">


However, everything does not work for MYRIAD (which required model in FP16 format).

benchmark_app MYRAD
<img width="599" alt="benchmarkONNXmyriad" src="https://user-images.githubusercontent.com/79643476/166405884-6270dc39-82b3-4fe8-b970-f42826aceacc.png">

Compile_tool MYRIAD
<img width="660" alt="MYRIADcompiletool" src="https://user-images.githubusercontent.com/79643476/166405934-cc982f31-37fb-41e1-9fd5-70138ac99a49.png">

IR conversion also failed with the error Mean/Scale value for images have different sizes: 3    2
**You may need to pay attention to this error.** This might the origin of your issue.
<img width="602" alt="IRfp16" src="https://user-images.githubusercontent.com/79643476/166406037-234b07f8-8571-4ef2-9b04-48aa1f8c42e1.png">

We'll investigate further to provide some more alternatives for you and we'll get back asap.

**Comment by avitial** [comment]
@friend045 I've taken a quick look at this issue and I see the shared ONNX file ([Sample onnx model](https://drive.google.com/drive/folders/1MfQ3qXMVG3jhwjmk9YBbmlYMI6OtOsVc?usp=sharing)) was produced with PyTorch v1.11.0, I was wondering if it would be possible to export using torch 1.8.1 (as listed in OMZ [requirements-pytorch.in](https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/model_tools/requirements-pytorch.in)) and see if the issue is still there on your side. 

@ORippler having issues accessing the model files shared, not sure if the link expired or files were removed. Could you also try my recommendation above and see if the problem is there?

**Comment by ORippler** [comment]
> @ORippler having issues accessing the model files shared, not sure if the link expired or files were removed. Could you also try my recommendation above and see if the problem is there?

Yeah the upload expired, reupped to google drive:
https://drive.google.com/file/d/1k06zr1tPhh5afSzgc7YMuHUXayudFfAt/view?usp=sharing

The model I want to convert cannot be exported with `PyTorch < v1.12.0`, since it uses `torch.cdist`, and support for exporting that to onnx was added in PyTorch v1.12.0

**Comment by DariaMityagina** [comment]
@friend045 , @ORippler hi! Could you please try applying this change https://github.com/openvinotoolkit/openvino/pull/14687?

**Comment by avitial** [comment]
PR has been merged, fix should be included in next 2022.3.1 LTS release. Closing this, feel free to reopen to ask any questions related to this topic.
